src/restoreImpl.ts

```diff
@@ -62,7 +62,7 @@ export async function restoreImpl(
                     ...restoreKeys
                 ].join(", ")}`
             );
-
+            core.setOutput(Outputs.CacheHit, false.toString());
```

Comment:
Should we set this output before the `if (failOnCacheMiss)` case above?

That would ensure it's set even if `fail-on-cache-miss` is true.

Reply:
Hi, @joshmgross 

Thanks for your response.

Indeed, since it applies even in the case of exceptions, it seems good to output before `failOnCacheMiss`. What do you think?
[6f5b5ee](https://github.com/actions/cache/pull/1404/commits/6f5b5ee7292482e1a430a02aa545bd3f446712fe)

Reply:
Thanks @fchimpan!

Looks like the `dist/` is out of date - could you re-run `npm run build`?

https://github.com/actions/cache/actions/runs/9599939755/job/26475812160?pr=1404

Reply:
Thanks @joshmgross !
[8addae5](https://github.com/actions/cache/pull/1404/commits/8addae5ffffa871c88fbd2d8a9a43af45f069161)
I have pushed the dist.